README.md

```diff
@@ -66,138 +80,106 @@ These tests can be run using `mvn verify`.
 Run trie stress test on cluster
 -------------------------------
 
-If you want to run the trie stress on the cluster, first set up HDFS, YARN, Zookeeper, 
-and Accumulo. Next, initialize Fluo with following configuration:
-```
-io.fluo.observer.0=io.fluo.stress.trie.NodeObserver
-io.fluo.app.trie.nodeSize=8
-io.fluo.app.trie.stopLevel=6
-```
+The [bin directory](/bin) contains a set of scripts to help run this test on a
+cluster.  These scripts make the following assumpitions.
 
-Next, build the module:
-```
-mvn package assembly:single
-```
+ * `FLUO_HOME` environment variable is set.  If not set, then set it in `conf/env.sh`.
+ * Hadoop `yarn` command is on path.
```

Comment:
Could also check for the existence of these commands in the scripts.  See http://stackoverflow.com/questions/592620/check-if-a-program-exists-from-a-bash-script